### DISCUSSION TITLE
my osu have a problem

### DISCUSSION THREAD

**Discussion by YAOSCHANGS** [ppy/osu#28393] [Q&A] [answered]
My osulazer

https://github.com/ppy/osu/assets/171758404/13eb56cc-276b-443c-9486-7abd55ae231e

(url) will flash after entering for a while, which makes me unable to play normally.

**Comment by Joehuu** [comment, marked as answer]
Judging by the black bars and graphical glitches, I'd assume you are on Android 14? Try updating Android to the latest version if not already. If that doesn't work, try the workaround described in https://github.com/ppy/osu/issues/25189#issuecomment-1821976470. Also tell what device you're using for tracking purposes, as Pixel devices already have the update that fixes this.
